package.json

```diff
@@ -102,12 +102,16 @@
     "sinon": "9.2.1",
     "subscriptions-transport-ws": "0.9.18",
     "supertest": "6.0.1",
-    "ts-node": "9.0.0",
-    "typescript": "4.1.2",
+    "ts-node": "10.0.0",
+    "typescript": "4.3.2",
     "unfetch": "4.2.0",
-    "ws": "5.2.2"
+    "ws": "7.4.6"
   },
   "peerDependencies": {
-    "graphql": "^14.7.0 || ^15.3.0"
+    "graphql": "^14.7.0 || ^15.5.0"
+  },
+  "optionalDependencies": {
+    "graphql-ws": "4.1.3",
+    "graphiql-subscriptions-fetcher": "0.0.2"
```

Comment:
Why are we listing these as optional dependencies? :thinking: 
It is unlikely that `npm` will fail to install them, so they will almost always be installed for the users of `express-graphql` (unless we expect users to run `npm install --no-optional`).
However, for those not using GraphQL subscriptions, these are two unnecessary dependencies. On the other hand, for those using subscriptions, it should be up to them to install these dependencies.

Reply:
Good point, they should be specified in the readme instesd